app-backend/api/src/main/scala/utils/QueryParameters.scala

```diff
@@ -69,4 +72,24 @@ trait QueryParametersCommon extends QueryParameterDeserializers {
   def shapeQueryParams = (
     orgQueryParams & userQueryParameters & timestampQueryParameters
   ).as(ShapeQueryParameters.apply _)
+
+  def searchQueryParams = parameters(
+    'searchPattern.as[String].?
+  ).as(SearchQueryParameters.apply _)
+
+  def toolParameters = parameters(
+    'minRating.as[Double].?,
+    'maxRating.as[Double].?,
+    'toolCategory.as[String].*,
+    'toolTag.as[String].*,
+    'search.as[String].?
```

Comment:
Is this being used? This would make it so that `search` and later on `searchPattern` are both valid query parameters. See comment above.

Reply:
Per offline discussion, this entire set of tool query parameters are deleted along with the case class since the implicit filters for them are not implemented anywhere at this stage. We may add them back if such needs arise in the future.